src/common/transformations/tests/common_optimizations/remove_multi_subgraph_op_dangling_params_tests.cpp

```diff
@@ -176,6 +176,11 @@ TEST_F(TransformationTestsF, RemoveLoopDanglingParametersIfConcatEmptyTensor) {
 }
 
 TEST_F(TransformationTestsF, RemoveIfDanglingParametersFromBodiesAndInputs) {
+#if defined(WIN32) && !defined(NDEBUG)
+    test_skipped = true;
+    GTEST_SKIP() << "Skipping smoke_Snippets_MHA3D_SplitM_withMul on Windows in Debug mode due to Issue 155258.";
```

Comment:
#27180 
If merged will solve this issue.

Reply:
reverted too